src/Pages/Main/Main.js

```diff
@@ -1,7 +1,138 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { BiSearch } from 'react-icons/bi';
+// import ReactDatePicker from 'react-datepicker';
+import 'react-dates/initialize';
+import {
+  DateRangePicker,
+  // SingleDatePicker,
+  // DayPickerRangeController,
+} from 'react-dates';
+import 'react-dates/lib/css/_datepicker.css';
 
 const Main = () => {
-  return <div>메인</div>;
+  // const [startdate, setStartdate] = useState(null);
+  // const [enddate, setEnddate] = useState(null);
+  const [date, setDate] = useState({
+    startdate: null,
+    enddate: null,
+  });
+
+  const [focusedInput, setfocusedInput] = useState(null);
+  const [person, setPerson] = useState(0);
+
+  const searchRoom = date => {
+    const moment = require('moment');
+    const a = moment(date.startdate).format('YYYY-MM-DD');
+    const b = moment(date.enddate).format('YYYY-MM-DD');
+    console.log(a);
+    console.log(b);
+    console.log(person);
+  };
+
+  const minNum = person => {
+    if (person > 0) {
+      setPerson(person - 1);
+    }
+  };
```

Comment:
```suggestion
  const subtractPerson = () => {
    if (person > 0) {
      setPerson(person - 1);
    }
  };
  
  const addPerson = () => {
    setPerson(person + 1);
  }
```
이렇게 함수를 작성하시면 onClick에 함수 이름만 넘겨주면 됩니다!
```jsx
onClick = {subtractPerson}
onClick = {addPerson}
```